Form/Extension/FormTooltipLabelTypeExtension.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * This file is part of the ElaoForm bundle.
+ *
+ * Copyright (C) 2014 Elao
+ *
+ * @author Elao <contact@elao.com>
+ */
+
+namespace Elao\Bundle\FormBundle\Form\Extension;
+
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * Extension for the FormType, provides a tooltip option
+ */
+class FormTooltipLabelTypeExtension extends AbstractTypeExtension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getExtendedType()
+    {
+        return 'form';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $resolver->setDefaults([
+            'tooltip_label'    => false,
+            'tooltip_position' => false,
+        ]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function finishView(FormView $view, FormInterface $form, array $options)
```

Comment:
You don't need to use the finishView here since you don't access child views.

(We should change method in `FormHelpTypeExtension.php` as well)